### PR TITLE
Build: create rails api structure with Rubocop, Rspec, Cors gems

### LINKED ISSUE #1: [1.0pt] - Set Up Structure - Back End Repo (Group Task)
-  Use correct Gitflow
- Create a new Ruby on Rails project for API by selecting `PostgreSQL` as the database and skipping the default test configuration (MiniTest): 
```
rails new final-capstone-back-end -d postgresql -T --api
```
- Create the database
- Install and set up linters (rubocop)
- Install and set up Rspec to test the app
- Install and set up rack-cors gem to provide support for Cross-Origin Resource Sharing (CORS)
- Add LICENSE file

### PR DESCRIPTION
**Pull request by HFG43**
Hi there! I'm sending this pull request for you to review my code. Thanks in advance for your feedback!

Milestones 🎯:
📌 Generate the Rails API structure
📌 Add Rubocop, Rspec and CORS Gems
📌 Install the gems running `bundle install`
📌 Configure CORS in `config/initializers/cors.rb` for GET, POST, PATCH, PUT & DELETE actions

🎖️ General requirements:
Gitflow
No linter errors
Ensure the use of best practices and Ruby style

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Hernán Güemes
+Copyright (c) 2023 Andy Zamorano && Andrea Manuel && Nitecelis Bravo Alcala && Hernán Güemes
```
Comment (NitBravoA92) [inline LICENSE:3]: @HFG43 I kindly suggest that you write the names of each team member correctly.
For example, nicknames should not be used in the LICENSE file (`Andy` should be changed to `Andrés`)
Likewise, the name `Nitecelis` is spelled incorrectly, it should be changed to: `Nitcelis`

**Review by NitBravoA92 — CHANGES_REQUESTED** [review]
## Hello @HFG43   , :wave:

Good job so far :+1: 
There are some issues that you still need to work on to go to the next task but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 
- Setup Ruby on Rails API project correctly :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NitBravoA92  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Review by NitBravoA92 — APPROVED** [review]
## STATUS: APPROVED :green_circle:  

Hi @HFG43   :wave: :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great Job implementing the suggested changes ✅ 

## Optional suggestions
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@NitBravoA92 ) in your question so I can receive the notification.

------

**Comment by HFG43** [comment]
@NitBravoA92 thanks for your feedback!!

**Pull request merged**